### ISSUE TITLE
Chrome Extension Not Available?

### ISSUE THREAD

**Issue by tnpir4002** [fcapano/Edit-This-Cookie#379]
I just set up a new computer and when Chrome imported all my extensions I noticed this one got skipped, and it's apparently not available from the Chrome Store anymore?

**Comment by Hacksore** [comment]
See my previous comment but I think the time has come from the big G 😞

https://github.com/ETCExtensions/Edit-This-Cookie/issues/370#issuecomment-2241356789

**Comment by MakcStudio** [comment]
The extension page right now is still not available, how soon should we expect the extension to come back to google?

**Comment by MakcStudio** [comment]
@fcapano read pls

**Comment by golong1989** [comment]
@fcapano 加油

**Comment by francosalcedo** [comment]
@fcapano

**Comment by mrfanii** [comment]
Hi,
I noticed your issue about this extension being unavailable in the Chrome Store. I’ve worked on a fork and released version 3, now available in the Chrome Web Store. 🎉

While I’ve kept the core code intact and made improvements, major updates can introduce issues. I’d really appreciate it if you could test it and share feedback or report any problems in the linked repo.

Looking forward to your thoughts!
[Edit-This-Cookie-V3](https://github.com/mrfanii/Edit-This-Cookie-V3)

**Comment by kreynen-du** [comment]
Not only did this get unpublished again, there is a copy cat that has actually copied the website too https://www.editthiscookies.com/

https://chromewebstore.google.com/detail/editthiscookies/hlgpnddmgbhkmilmcnejaibhmoiljhhb/reviews

**Comment by YoeriW** [comment]
I've also made a fork and released it to the Chrome Web Store because I wasn't sure about 'EditThisCookies'; the website seems very sketchy.

What I've done:

- Updated for Manifest V3
- Updated jQuery, jQueryUI, and Font Awesome
- Added all pull requests to this project
- Added a dark theme
- Fixed some bugs

But I am open to new requests as well. As long as the original owner does not update the extension, I will keep updating the fork.

https://github.com/YoeriW/Edit-This-Cookie
https://chromewebstore.google.com/detail/editthiscookie/ihfmcbadakjehneaijebhpogkegajgnk

**Comment by mobeigi** [comment]
> Not only did this get unpublished again, there is a copy cat that has actually copied the website too https://www.editthiscookies.com/
> 
> https://chromewebstore.google.com/detail/editthiscookies/hlgpnddmgbhkmilmcnejaibhmoiljhhb/reviews

This one is apparently Chinese spyware and tracks you. Avoid using this one at all costs.

**Comment by Hacksore** [comment]
> > Not only did this get unpublished again, there is a copy cat that has actually copied the website too https://www.editthiscookies.com/
> > https://chromewebstore.google.com/detail/editthiscookies/hlgpnddmgbhkmilmcnejaibhmoiljhhb/reviews
> 
> This one is apparently Chinese spyware and tracks you. Avoid using this one at all costs.

Yeah there was an analysis video done by @atericparker on this new extension.

https://youtu.be/AZgM_50aqbg

**Comment by JohnCanny52** [comment]
> > > Not only did this get unpublished again, there is a copy cat that has actually copied the website too https://www.editthiscookies.com/
> > > https://chromewebstore.google.com/detail/editthiscookies/hlgpnddmgbhkmilmcnejaibhmoiljhhb/reviews
> > 
> > 
> > This one is apparently Chinese spyware and tracks you. Avoid using this one at all costs.
> 
> Yeah there was an analysis video done by @atericparker on this new extension.
> 
> https://youtu.be/AZgM_50aqbg

This fake extension inserts an "AI Suggestions" ad bar at the top of the page, mimicking the Chrome UI. I almost mistook it for a new, annoying feature developed by Google.

**Comment by atericparker** [comment]
From the reviews looks like they just added that "Feature"

**Comment by zelenooki87** [comment]
Chrome blocked new extension and flagged it as malware
`https://www.editthiscookies.com/
https://chromewebstore.google.com/detail/editthiscookies/hlgpnddmgbhkmilmcnejaibhmoiljhhb/reviews`

Do I need to change my password on all sites now.... WTH!!!

**Comment by atericparker** [comment]
Frustrating it took them weeks to block it. I would advise that, but in terms of observed behavior the main concern is just a privacy one, it logs every site you visit and your IP address. 

I got into the backend here: https://www.youtube.com/watch?v=6CEEKPrw4MA
